runtime/vm/jvminit.c

```diff
@@ -154,6 +154,10 @@ typedef struct {
 	char* j9libvmDirectory;
 } J9InitializeJavaVMArgs;
 
+#if defined(WIN32) && defined(_WIN32_WINNT_WINBLUE) && (_WIN32_WINNT_MAXVER >= _WIN32_WINNT_WINBLUE)
+typedef BOOL (WINAPI * PMP_func)(HANDLE, PROCESS_MITIGATION_POLICY, PVOID, SIZE_T);
```

Comment:
Line 158 could be immediately after `#include <VersionHelpers.h>` so we don't need to repeat the conditions.
Alternatively it can be immediately before line the use on line 8108.

Reply:
Addressed in [f487b7d](https://github.com/eclipse-openj9/openj9/commit/f487b7debac3f68af165f4b3c951f67811140d65)